workers/unity/Assets/Playground/Config/CubeTemplate.cs

```diff
@@ -0,0 +1,44 @@
+using Generated.Improbable;
+using Generated.Improbable.Transform;
+using Generated.Playground;
+using Improbable.Gdk.Core;
+using Improbable.Worker;
+using Improbable.Worker.Core;
+
+namespace Playground
+{
+    public static class CubeTemplate
+    {
+        public static Entity CreateCubeEntityTemplate(Coordinates coords)
+        {
+            const string entityType = "Cube";
+
+            var transform = Transform.Component.CreateSchemaComponentData(
+                new Location((float) coords.X, (float) coords.Y, (float) coords.Z),
+                new Quaternion(1, 0, 0, 0),
+                0
+            );
+
+            var cubeColor = CubeColor.Component.CreateSchemaComponentData();
+            var cubeTargetVelocity = CubeTargetVelocity.Component.CreateSchemaComponentData(new Vector3f { X = -2.0f });
```

Comment:
Out of curiosity, what is the `X = -2.0f` for?

Reply:
This was transplanted from the snapshot code, I'm guessing by default the cubes move left when they spawn